monkey/infection_monkey/exploit/HostExploiter.py

```diff
@@ -59,10 +59,20 @@ def send_exploit_telemetry(self, name: str, result: bool):
             attempts=self.exploit_attempts,
         ).send()
 
-    def report_login_attempt(self, result, user, password="", lm_hash="", ntlm_hash="", ssh_key=""):
+    def report_login_attempt(
+        self,
+        exploitation_result,
+        propagation_result,
+        user,
+        password="",
+        lm_hash="",
+        ntlm_hash="",
+        ssh_key="",
+    ):
```

Comment:
This is stale code. `send_exploit_telemetry` (which uses `self.exploit_attempts`) doesn't do anything so calling this function is pointless.

Reply:
Even if it did, `exploitation_result` and `propagation_result` shouldn't be part of `exploit_attempts`.